integrated_channels/canvas/exporters/content_metadata.py

```diff
@@ -52,6 +70,12 @@ def transform_description(self, content_metadata_item):
                 base_description=base_description, short_description=short_description
             )
 
+        formatted_start_date = convert_date_str(content_metadata_item.get('start', 'N/A'))
+        formatted_end_date = convert_date_str(content_metadata_item.get('end', 'N/A'))
+        description = (f"{description} <br />"
```

Comment:
Is the course description able to be customized by creators? Is this start and end date enforced, or can it be taken out if they want to put something else in there?

Reply:
good q. Right now we are enforcing this, meaning, even if they change on their side, if there is a future update we send over, it will overwrite what is there. So think of it like a managed metadata (controlled by edX)